From bd10ec336ee5e7b1317c77f03b5136cea79fa91b Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Fri, 22 Mar 2013 20:47:34 -0700
Subject: [PATCH 05/10] ck: remove expert designation from memory split:

Imported from: http://ck.kolivas.org/patches/3.0/3.8/3.8-ck1/patches/kconfig-expose_vmsplit_option.patch

The options to alter the vmsplit to enable more lowmem are hidden behind the
expert option. Make it more exposed for -ck users and make the help menu
more explicit about what each option means.

-ck

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 arch/x86/Kconfig |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index 0694d09..8b64d9d 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -1108,7 +1108,7 @@ endchoice
 
 choice
 	depends on EXPERIMENTAL
-	prompt "Memory split" if EXPERT
+	prompt "Memory split"
 	default VMSPLIT_3G
 	depends on X86_32
 	---help---
@@ -1128,17 +1128,17 @@ choice
 	  option alone!
 
 	config VMSPLIT_3G
-		bool "3G/1G user/kernel split"
+		bool "Default 896MB lowmem (3G/1G user/kernel split)"
 	config VMSPLIT_3G_OPT
 		depends on !X86_PAE
-		bool "3G/1G user/kernel split (for full 1G low memory)"
+		bool "1GB lowmem (3G/1G user/kernel split)"
 	config VMSPLIT_2G
-		bool "2G/2G user/kernel split"
+		bool "2GB lowmem (2G/2G user/kernel split)"
 	config VMSPLIT_2G_OPT
 		depends on !X86_PAE
-		bool "2G/2G user/kernel split (for full 2G low memory)"
+		bool "2GB lowmem (2G/2G user/kernel split)"
 	config VMSPLIT_1G
-		bool "1G/3G user/kernel split"
+		bool "3GB lowmem (1G/3G user/kernel split)"
 endchoice
 
 config PAGE_OFFSET
-- 
1.7.10.4

